### ISSUE TITLE
Project crashes if horizontal size is UIUserInterfaceSizeClass.regular

### ISSUE THREAD

**Issue by JagCesar** [apple/cloudkit-sample-sharing#10]
When trying the sample project on an iPad the app crashes if I do the following on an iPad and the `horizontalSize` is `.regular`.

1. Create a contact to share
2. Refresh to show contact
3. Tap Share button
4. Select iMessage icon

Expected result:
- iMessage view should be presented

Actual result:
- App crashes with the following log.

```
2021-07-04 10:12:34.305876+0200 Sharing[1668:1436276] [Assert] Setting modalPresentationStyle once presentationController has been accessed will have no effect until <UICloudSharingController: 0x15405ce00> is presented, dismissed, and presented again.
2021-07-04 10:12:34.308316+0200 Sharing[1668:1436276] *** Terminating app due to uncaught exception 'NSGenericException', reason: 'UIPopoverPresentationController (<_UISharingViewPresentationController: 0x15510fcc0>) should have a non-nil sourceView or barButtonItem set before the presentation occurs.'
*** First throw call stack:
(0x191627754 0x1a60ee7a8 0x1937c8bb8 0x1937d48d0 0x1937d2228 0x1944f2c98 0x1940256c4 0x19401403c 0x194047f10 0x1915a05e0 0x19159a704 0x19159acb0 0x19159a360 0x1a8bd8734 0x194015584 0x19401adf4 0x1a535e24c 0x102736270 0x1027361e8 0x102736308 0x191256cf8)
libc++abi: terminating with uncaught exception of type NSException
*** Terminating app due to uncaught exception 'NSGenericException', reason: 'UIPopoverPresentationController (<_UISharingViewPresentationController: 0x15510fcc0>) should have a non-nil sourceView or barButtonItem set before the presentation occurs.'
terminating with uncaught exception of type NSException
```

**Comment by JagCesar** [comment]
The issue seems to be that the `UICloudSharingController` instance created in `CloudSharingView` doesn't have a `sourceView`. I tried setting the `sourceView` to new view with a frame of `.zero` as a temporary solution.

`sharingController.popoverPresentationController?.sourceView = UIView(frame: CGRect.zero)`

It no longer crashes, but it's not very elegant either 😅 I think a proper solution would be to use the Share button as source view, but maybe that adds too much complexity to the example?

**Comment by sjrmanning** [comment]
Out of curiosity, is this happening on the latest commit? The fix for #1 should have fixed CloudSharingView requiring a `sourceView`.

**Comment by JagCesar** [comment]
My bad, you're completely right. Yes, this is fixed in main. 🤦🏾‍♂️
